.buildkite/metricbeat/pipeline.yml

```diff
@@ -264,27 +209,7 @@ steps:
           - "metricbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "metricbeat: Extended MacOS x86_64 Unit Tests"
-
-      - label: ":mac: MacOS arm64 Unit Tests"
-        key: "extended-macos-arm64-unit-tests"
-        skip: "due to https://github.com/elastic/beats/issues/33035"
-        command: |
-          set -euo pipefail
-          source .buildkite/scripts/install_macos_tools.sh
-          cd metricbeat && mage build unitTest
-        retry:
-          automatic:
-            - limit: 3
-        agents:
-          provider: "orka"
-          imagePrefix: "${IMAGE_MACOS_ARM}"
-        artifact_paths:
-          - "metricbeat/build/*.xml"
-          - "metricbeat/build/*.json"
-        notify:
-          - github_commit_status:
-              context: "metricbeat: Extended MacOS arm64 Unit Tests"
+              context: "metricbeat: EmacOS x86_64 Unit Tests"
```

Comment:
```suggestion
              context: "metricbeat: macOS x86_64 Unit Tests"
```

Reply:
Sorry, there was auto-merge enabled, will to a tiny PR to fix typo